SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestReplicatedStartupActor/SpatialTestReplicatedStartupActor.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "SpatialTestReplicatedStartupActor.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "Kismet/GameplayStatics.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "Net/UnrealNetwork.h"
+#include "ReplicatedStartupActorPlayerController.h"
+
+
+/**
+ * This test automates the ReplicatedStartupActor gym. The gym was used for:
+ * - QA workflows Test Replicated startup actor are correctly spawned on all clients
+ * - To support QA test case "C1944 Replicated startup actors are correctly spawned on all clients"
+ * NOTE: This test requires a specific Map, with a custom GameMode and PlayerController, trying to run this test on a different Map will make it fail.
```

Comment:
I would mention that the test requires the ReplicatedStartupActor placed in the map as well.